doc/api/cli.md

```diff
@@ -1491,6 +1499,16 @@ Disable [runtime allocation of executable memory][jitless]. This may be
 required on some platforms for security reasons. It can also reduce attack
 surface on other platforms, but the performance impact may be severe.
 
+### `--localstorage-file=file`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+The file used to store `localStorage` data. If the file does not exist, it is
+created the first time `localStorage` is accessed. This flag is a no-op unless
+Node.js is started with the `--experimental-webstorage` flag.
```

Comment:
Should we document whether it's possible and/or safe to use the same file from two or more processes running concurrently?